build_sysroot/prepare_sysroot_src.sh

```diff
@@ -20,6 +20,13 @@ git init
 echo "[GIT] add"
 git add .
 echo "[GIT] commit"
+
+# This is needed on virgin system where nothing is configured.
+# git really needs something here, or it will fail.
+# Even using --author is not enough.
+git config user.email || git config user.email "none@none.com"
```

Comment:
```suggestion
git config user.email || git config user.email "info@example.com"
```
none.com is an existing domain. example.com is reserved for example purposes.

Reply:
Didn't know about example.com being reserved for this, thanks!

Reply:
It has been reserved by [RFC2606](https://datatracker.ietf.org/doc/html/rfc2606).

Reply:
TIL !